index.html

```diff
@@ -43,23 +43,20 @@
         <dl class="info-headings">
           <div>
             <dt>Data Gathered Since</dt>
-              <!-- @todo get date-start from store -->
-            <dd id="date-start">
-              <time datetime="2017-06-13T11:00">June 13, 2017</time>
+            <dd>
+              <time id="data-gathered-since" datetime=""></time>
             </dd>
           </div>
           <div>
             <dt>You Have Visited</dt>
-            <!-- @todo get num-sites-visited from store -->
             <dd>
-              <var id="num-sites-visited">6</var> Sites
+              <var id="num-first-parties"></var>
             </dd>

```

Comment:
Because we don't have a date value until they visit a site, so we'd have inconsistent entries (no value for date, a zero value for the other two). Honestly, I don't care either way, just saying I had it the way you suggest before, and I'm more inclined this way. We'll let @jonathanKingston decide I suppose!